docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -2,7 +2,7 @@
 
 ## 10.0.11
 
-See the [announcement](http://akka.io/blog/news/2017/12/01/akka-http-10.0.11-released.html) and
+See the [announcement](http://akka.io/blog/news/2017/12/01/akka-http-10.0.11-released) and
```

Comment:
Hmm this URL works for me both with and without `.html`? Would be nice to link to https btw

Reply:
I go to the page and click the link it fails: https://doc.akka.io/docs/akka-http/current/scala/http/release-notes.html

Reply:
Wait, I think I know what happens, @jonas already fixed it, but I didn't realize!

Reply:
I can upgrade the link to https though :)

Reply:
This link has already been fixed to go to `/blog/news/` instead of `/news`. The `/news` path is old and gets redirected to the new location but that doesn't seem to work for the `.html` version.

Reply:
I realised that now! sorry